Client.xcodeproj/project.pbxproj

```diff
@@ -10535,7 +10534,6 @@
 				C8E2E80E23D20FD2005AACE6 /* FxAWebViewController.swift in Sources */,
 				E14F7DF2288F3F9F00E3722C /* ThemedTableSectionHeaderFooterView.swift in Sources */,
 				E1442FBF294782B6003680B0 /* CGRect+Extension.swift in Sources */,
-				8AB8574427D963960075C173 /* Notifiable.swift in Sources */,
```

Comment:
Notifiable still exists in utils but is gone from pbxproj? 

Am I reading this right or only got removed from parts of pbxproj 🤔

Reply:
I cloned the project and this is more of a cleanup right?

Reply:
It should be gone completely from the project and now lives in BrowserKit.

Reply:
<img width="357" alt="image" src="https://user-images.githubusercontent.com/8919439/210010813-4db19b8d-f54c-4743-84df-d7a81e0ca4e3.png">

Perfect, had to cleanup XCode but have the one and only instance